blueprints/node-webkit/index.js

```diff
@@ -50,5 +51,20 @@ module.exports = {
         return writeFile(packageJsonPath, JSON.stringify(json, null, '  '));
       }
     });
+  },
+  addQUnitLogger: function(){
+    var testHelperFile = path.resolve(this.project.root, 'tests/test-helper.js');
+    var _this = this;
+    
+    return readFile(testHelperFile, { encoding: 'utf8'} ).then(function(data){
+      var code = 'require(\'' + _this.project.name() + '/tests/helpers/qunit-logger\');';
```

Comment:
This is a bit weird. You end up having an AMD `require` call inside of an ES6 module. You can keep `qunit-logger.js` as is, but I would move it to something like `vendor/node-webkit/qunit-logger.js` and merge it together with the `test-support` tree using the addon build hooks.